pylint_odoo/checkers/no_modules.py

```diff
@@ -213,6 +213,13 @@
     ),
 })
 
+ODOO_MSGS.update({
+    'W%d12' % settings.BASE_NOMODULE_ID: (
+        'Eval referenced.',
```

Comment:
eval is a method name, then we should use without uppercase transformation.
What about `"eval" referenced`?